.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -99,17 +98,28 @@ body:
       label: Hotfix release checklist
       description: Complete these items in sequence as the `Hotfix` release progresses
       options:
-        - label: Make the hotfix in a branch from `prod`
-        - label: Bump the application version
+        - label: Create a hotfix branch from the latest release tag on `main` (`git checkout -b hotfix YYYY.0M.R`)
+        - label: Commit the fixes to the hotfix branch
         - label: Ensure `prod` secrets are up to date
-        - label: Open a PR from the hotfix branch into `prod`, merge
+        - label: Create an annotated tag on the hotfix branch (`git tag -a YYYY.0M.R`)
+        - label: Push the annotated tag (`git push origin YYYY.0M.R`)
         - label: Smoke Test the fix in prod
         - label: Confirm acceptance of Smoke Tests by adding a comment to this issue
-        - label: Tag the release on `prod`, push the tag to GitHub (see [docs](https://docs.calitp.org/benefits/deployment/release/#5-tag-the-release) for commands)
-        - label: Create a release in GitHub for the tag, generating release notes
         - label: Edit release notes to link back to this Release process issue
         - label: Edit release notes with additional context, images, animations, etc. as-needed
-        - label: Create a branch off `main`
-        - label: Open a PR from `prod` into that branch, merge
-        - label: (optional) Adapt hotfix to current state of `main` in that same branch
+        - label: Open a PR from the hotfix branch to the current state of `main`
         - label: Merge the PR into `main`
+  - type: checkboxes
+    id: rollback-checklist
+    attributes:
+      label: Rollback release checklist
+      description: Complete these items in sequence as the `Rollback` release progresses
+      options:
+        - label: Create a rollback branch from the last known good release tag on `main` (`git checkout -b rollback YYYY.0M.R`)
```

Comment:
I thought we decided we don't need this?

Reply:
Sorry, I misunderstood our previous conversation. Just to make sure, does this look correct?

- Create an annotated release tag for the rollback on the commit associated with the last known good release tag on `main` (`git tag -a YYYY.0M.R <commit-hash>`)
- Ensure `prod` secrets are up to date
- Push the annotated tag (`git push origin YYYY.0M.R`)
- Smoke Test the rollback in prod
- Confirm acceptance of Smoke Tests by adding a comment to this issue
- Edit release notes to link back to this Release process issue
- Edit release notes with additional context, images, animations, etc. as-needed

Reply:
Yeah sorry I wasn't clear, I was in agreement with your assessment:

> I am updating the checklist and realized that we probably don't even need to initially create a `rollback` branch

Reply:
That list looks right :+1:

Reply:
Oh no problem, perfect, thank you!